Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:20-slim
+
+WORKDIR /opt/app
+
+COPY ./package.json .
+COPY ./package-lock.json .
+RUN npm install
+
+COPY ./scripts ./scripts
+COPY ./src ./src
+COPY .env .
+
+RUN useradd -m githubApp
+
+USER githubApp
+
+CMD [ "npm", "run", "dev"]
```

Comment:
Is this meant for local development only? Because the server is not suited for production deployment, it's tailored for local dev only

Reply:
It helps me in fast setup for local dev without having to install things I may not need later on my laptop

Reply:
all instals are local, so as long as you have Node you are good, there is no polluting in other places.